src/auditlog_tests/test_settings.py

```diff
@@ -30,14 +30,6 @@
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': 'auditlog_tests.db',
     },
-    'postgres': {
```

Comment:
why did you remove the postgres database tests? We have some tests that test postgres functionality specifically, I can understand that when you ran locally you might not have had a postgres server / db setup, but we shouldn't be removing it

Reply:
Sorry, I did not see that I committed that file as well.
I removed it for local testing on my end.
Do you want me to submit a complete new PR?